src/issue-strategies/index.js

```diff
@@ -1,17 +1,15 @@
 import * as bugMtStrat from './bug-maintenance.js';
-import * as initStrat from './initiative.js';
-import * as epicStrat from './epic.js';
 import * as storyStrat from './story.js';
-import * as taskStrat from './task.js';
+import * as notAllowedStrat from './not-allowed.js';
 
 export default {
-  Initiative: initStrat,
-  Epic: epicStrat,
+  Initiative: notAllowedStrat,
+  Epic: notAllowedStrat,
```

Comment:
That would be a change here: https://github.com/TWExchangeSolutions/jira-precommit-hook/blob/da7a5d7b717df6312bba26de5cf83fab651c37d8/src/validation-strategies/one-valid-issue.js#L7-L9